azkaban-common/src/main/java/azkaban/executor/container/ContainerizedImpl.java

```diff
@@ -20,4 +20,5 @@
 public interface ContainerizedImpl {
   void createContainer(final int executionId) throws ExecutorManagerException;
   void deleteContainer(final int executionId) throws ExecutorManagerException;
+  void deleteContainers(final long containerValidity) throws ExecutorManagerException;
```

Comment:
1. would it be helpful if the signature is 
deleteContainers(10, TimeUnit.Days)? 
the method can then do `timeUnit.toMillis(duration)`
2. would it make sense if we add the aged deletion part to the method name? for example, `deleteAgedContainers`?

Reply:
Thanks for reviewing. 
1. Changed to Duration from long
2. Renamed the method to deleteAgedContainers
3. This is much more convenient.